src/rusticTheme.ts

```diff
@@ -169,6 +169,15 @@ let rusticLightTheme = createTheme({
         },
       },
     },
+    MuiTextField: {
+      styleOverrides: {
+        root: {
+          '& .MuiInputBase-root': {
+            backgroundColor: whiteColor,
+          },
+        },
+      },
+    },
```

Comment:
This is what it defaults to. I dont think this is required
```suggestion
```

Reply:
Mui Textfield component has no default background colour. If I add to a div with colour I get this result:

![Screenshot 2024-04-03 at 2 17 27 PM](https://github.com/rustic-ai/ui-components/assets/111031789/17e55102-bd93-49e6-995c-3f4998191418)

Reply:
I think that is ok and expected. We or the user could add another variant or overwrite the class if so desired.